client/my-sites/media-library/list-item.jsx

```diff
@@ -143,3 +147,10 @@ export default class extends React.Component {
 		);
 	}
 }
+
+export default connect( state => {
+	return {
+		isContributor: ! canCurrentUser( state, getSelectedSiteId( state ), 'publish_posts' ),
+		isMediaLibrary: getSectionName( state ) === 'media',
+	};
+} )( MediaLibraryListItem );
```

Comment:
Got following warning:

<img width="983" alt="screenshot_872" src="https://user-images.githubusercontent.com/127594/63467407-4528b400-c41a-11e9-8542-c067479630a7.png">

Adding `{}` or `() => {}` as `mapDispatchToProps` cleared the warning.

Reply:
Thanks, I completely missed those errors!

I'm not sure if I did this correctly but I took a look at some examples in the rest of Calypso and I think I've managed to clear the console errors now.